README.md

```diff
@@ -1,26 +1,36 @@
-# MoUSE
+# MoUSE backend overview
 
 **MoUSE** - **Mo**use **U**ltrasonic **S**queak **E**xplorer
 
-Toolkit for processing, localisation and classification of rodent ultrasonic squeaks written in Python.
+This repo contains the backend for our whole JosephTheMoUSE project. This is toolkit for processing, localisation and classification of rodent ultrasonic squeaks written in Python. 
 
-I you want to install MoUSE Desktop App see [MoUSE Desktop App repository](https://github.com/JosephTheMoUSE/MoUSE-GUI#mouse-desktop-app). If you are looking for a MoUSE Desktop App usage tutorial,
-see [MoUSE wiki page](https://github.com/JosephTheMoUSE/MoUSE-docs/wiki).
+## Backend MoUSE istallation 
 
-## Install
+#### For Developers
 
-#### Developers
+If you are familiar with programming, use below instructions for this MoUSE backend installation. 
+Firstly, it is available to run it on windows/linux/macOS. You just need to have installed on your computer Python in version ^3.11 and Poetry compatible with your Python version. In the Python installation add it to the environment variable PATH on your computer. The same do with the Poetry. Add it to the environment variable PATH. We are using Poetry for dependency management as it is nowadays very popular and efficient tool for this purpose.
+
+Now, please clone this repository to your destination folder as follows:
 
-Use this commands to create link to this package in python's `site-packages`:
 ```bash
 $ git clone https://github.com/JosephTheMoUSE/MoUSE.git
 $ cd MoUSE
-$ python -m pip install -e .
 ```
-All changes will be available without reinstallation (`site-packages` for this package just points to `src` directory).
+Then, open it in your favourite Python IDE. We are using Pycharm as it is very efficient and common tool for programming in Python. If you open our project in the IDE, you just need to put in your terminal (in the main project folder) command:
```

Comment:
Do not enforce users to use any IDE, in fact the installation instruction should be IDE-independent. Do not mention IDE here :)